jss.spec

```diff
@@ -108,13 +108,26 @@ export CFLAGS
 # Check if we're in FIPS mode
 modutil -dbdir /etc/pki/nssdb -chkfips true | grep -q enabled && export FIPS_ENABLED=1
 
+# RHEL's CMake doesn't support -B flag.
+%if 0%{?rhel}
+%{__mkdir_p} %{_vpath_builddir}
+cd %{_vpath_builddir}
+%endif
```

Comment:
This seems to be opposite to one in dogtagpki/pki. That is, we ask to use `_vpath_builddir` in Fedora and `build` in RHEL. 

Here is the code from PKI repo: https://github.com/dogtagpki/pki/pull/554/files#diff-309669ccd4cb6f6fdf531483d210a34dR833-R836

Reply:
Endi wrote the one in PKI and kept it at `build/` in RHEL. In JSS, since `%{_vpath_builddir}` is defined on RHEL, I kept it, and only stubbed around the missing CMake functionality.

Reply:
(in JSS it is harder to keep `build/` vs `%{_vpath_builddir}` since we use the latter in the `%install` section.)